### PR TITLE
feat: fetaure email identities.

### PR DESCRIPTION
**Pull request by nileshgadgi**
## what
* Feature E-Mail identities in AWS SES.
* Remove unnecessary variables.
* Add and updated the required changes in the workflows.

## why
* To enhance the module usability and accessibility for ease of use.

## references
* https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ses_email_identity

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -29,12 +29,19 @@ resource "aws_ses_domain_identity" "default" {
   domain = var.domain
 }
 
+#Module      : EMAIL IDENTITY
+#Description : Terraform module to create Emails identity using domain
+resource "aws_ses_email_identity" "default" {
+  count = var.enable_email ? length(var.emails) : 0
```
Comment (nikitadugar) [inline main.tf:35]: missing var.enabled here in condition

File: main.tf [hunk 2]
```diff
@@ -55,6 +62,7 @@ resource "aws_route53_record" "ses_verification" {
 # Module      : DOMAIN DKIM
 # Description : Terraform module which creates Domain DKIM resource on AWS
 resource "aws_ses_domain_dkim" "default" {
+  count  = var.enable_domain ? 1 : 0
```
Comment (nikitadugar) [inline main.tf:65]: same var.enabled missing here

**Review by nikitadugar — CHANGES_REQUESTED** [review]

File: main.tf (hunk 1 above)
Comment (nileshgadgi) [reply main.tf]: Fixed. 👍🏼

File: main.tf (hunk 2 above)
Comment (nileshgadgi) [reply main.tf]: Fixed. 👍🏼

**Review by nikitadugar — APPROVED** [review]

**Review by themaniskshah — APPROVED** [review]
LGTM!

**Pull request merged**